SPECS/cracklib/cracklib.spec

```diff
@@ -1,4 +1,3 @@
-%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 %{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
```

Comment:
Nit: While we're here, we can go ahead and remove the defines for `%python3_sitelib` and similar, since we define them [here](https://github.com/microsoft/CBL-Mariner/blob/1f0a8ae9fa285384aee0465a1e2aa7b9c0ec4ed4/SPECS/mariner-rpm-macros/macros.python3#L1-L2)